src/main/java/com/kodilla/ecommercee/CartController/CartController.java

```diff
@@ -0,0 +1,48 @@
+package com.kodilla.ecommercee.CartController;
+
+import com.kodilla.ecommercee.CartController.CartDomainDto.CartDto;
+import com.kodilla.ecommercee.CartController.CartDomainDto.CartItemDto;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.function.Function;
+
+@RestController
+@CrossOrigin("*")
+@RequestMapping("v1/carts")
+@RequiredArgsConstructor
+public class CartController {
+
+    @GetMapping("/cart")
+    public List<CartItemDto> getCart() {
+        CartItemDto cartItem1 = new CartItemDto(1L, "Kurtka", "Kurtka elegancka jesienna", 750, 2);
+        CartItemDto cartItem2 = new CartItemDto(2L, "Kurtka", "Kurtka sportowa jesienna", 850, 2);
+        CartItemDto cartItem3 = new CartItemDto(3L, "Kurtka", "Kurtka zimowa narciarska", 1200, 5);
+        List<CartItemDto> cart = new ArrayList<>();
+        cart.add(cartItem1);
+        cart.add(cartItem2);
+        cart.add(cartItem3);
+
+        return cart;
+    }
+
+    @PostMapping
+    public ResponseEntity<Void> addCartItem(@RequestBody CartItemDto cartItemDto) {
+        return ResponseEntity.ok().build();
+    }
+
+    @DeleteMapping(value = {"cartItemId"})
+    public ResponseEntity<Void> removeCartItem(@PathVariable Long cartItemId) {
+        return ResponseEntity.ok().build();
+    }
+
+    @GetMapping("/order")
+    public ResponseEntity<CartDto> createOrder() {
+        //I don't know what to put in here. Need ideas ;)//
```

Comment:
I think that at the end here will be something like OrderService.save(CartDto) or similar but now imho it can stay like this.

Reply:
I think you have to change @GetMapping to @PostMapping though.

Reply:
Ok. Thank you.